tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
+
+_CppAudioBuffer = _pywrap_audio_buffer.AudioBuffer
+_CppAudioFormat = _pywrap_audio_buffer.AudioFormat
+_LoadAudioBufferFromFile = _pywrap_audio_buffer.LoadAudioBufferFromFile
+
+
+class TensorAudio(object):
+  """A wrapper class to store the input audio."""
+
+  def __init__(self,
+               audio_format: _CppAudioFormat,
+               sample_count: int,
+               audio_data: _CppAudioBuffer = None,
+               is_from_file: bool = False,
+               ) -> None:
+    """Initializes the `TensorAudio` object.
+
+    Args:
+      audio_format: C++ AudioFormat object, format of the audio.
+      sample_count: int, number of samples in the audio.
+      audio_data: C++ AudioBuffer object, contains raw audio data, buffer size
+      and audio format info.
+      is_from_file: boolean, whether `audio_data` is loaded from the audio file.
+    """
+    self._format = audio_format
+    self._sample_count = sample_count
+    self._is_from_file = is_from_file
+
+    if self._is_from_file:
+      self._data = audio_data
+    else:
+      self.clear()
+
+  def clear(self):
+    """Clear the internal buffer and fill it with zeros."""
+    self._buffer = np.zeros(
```

Comment:
@khanhlvg FYI, the tests seem to work fine without a contiguous NumPy array so I'm reverting the change from my previous commit. I believe the issue was leaving the datatype unspecified.